ui/addon/src/main/java/org/jboss/windup/ui/RulesetUpdateChecker.java

```diff
@@ -54,8 +57,7 @@ public void perform(@Observes PostStartup event)
         }
     }
 
-    public static boolean rulesetNeedUpdate(DependencyResolver dependencyResolver)
-    {
+    public static boolean rulesetNeedUpdate(DependencyResolver dependencyResolver, String rulesPomXml) {
         List<Coordinate> resolveVersions = dependencyResolver.resolveVersions(DependencyQueryBuilder.create(CoordinateBuilder.create()
```

Comment:
Same for any other places that require internet connectivity, but this is the most important because it will be run automatically afaik.